packages/mobile/src/identity/verification.ts

```diff
@@ -63,23 +63,31 @@ export function* startVerification() {
 
   Logger.debug(TAG, 'Starting verification')
 
+  const startTime = Date.now()
+
   const { result, cancel, timeout } = yield race({
     result: call(doVerificationFlow),
     cancel: take(Actions.CANCEL_VERIFICATION),
     timeout: delay(VERIFICATION_TIMEOUT),
   })
 
   if (result === true) {
-    CeloAnalytics.track(CustomEventNames.verification_success)
+    ValoraAnalytics.track(AnalyticsEvents.verification_success, {
+      duration: Date.now() - startTime,
+    })
     Logger.debug(TAG, 'Verification completed successfully')
   } else if (result === false) {
-    CeloAnalytics.track(CustomEventNames.verification_failed)
+    ValoraAnalytics.track(AnalyticsEvents.verification_failed, { duration: Date.now() - startTime })
     Logger.debug(TAG, 'Verification failed')
   } else if (cancel) {
-    CeloAnalytics.track(CustomEventNames.verification_cancelled)
+    ValoraAnalytics.track(AnalyticsEvents.verification_cancelled, {
+      duration: Date.now() - startTime,
+    })
     Logger.debug(TAG, 'Verification cancelled')
   } else if (timeout) {
-    CeloAnalytics.track(CustomEventNames.verification_timed_out)
+    ValoraAnalytics.track(AnalyticsEvents.verification_timed_out, {
+      duration: Date.now() - startTime,
```

Comment:
Do these durations end up passed as a string? Not sure how the pipeline handles date objects

Reply:
`Date.now` returns a number

Reply:
Yup, it's a number type